debussy_concert/reverse_etl/config/movement_parameters/reverse_etl.py

```diff
@@ -51,11 +49,12 @@ class ReverseEtlMovementParameters(MovementParametersBase):
     reverse_etl_query: str
     reverse_etl_dataset_partition_type: str
     reverse_etl_dataset_partition_field: str
-    extraction_query_from_temp: str
     destination_type: str
     output_config: OutputConfig
-    destination_object_path: str
     destination_connection_id: str
+    extraction_query_from_temp: Optional[str] = None
+    destination_object_path: Optional[str] = None
+    destination_table: Optional[str] = None
 
```

Comment:
those should not be optional

Reply:
remove destination_table attribute for to use destination_object_path and update to required field
s extraction_query_from_temp and destination_object_path